app/assets/stylesheets/ppm-estimator.print.scss

```diff
@@ -0,0 +1,21 @@
+@import "application.scss";
+
+header, h1, h4, footer, .breadcrumbs, .login-banner, .back-to-top, .ppm-description, .ppm-form, .ppm-estimate-print {
+  display: none;
+}
+
+.ppm-estimate-results {
+  border-top: 0;
+  margin-top: 0;
+  padding-top: 0;
+}
+
+.usa-section {
+  padding: 0;
+}
+
+@media print {
```

Comment:
Since this is a print stylesheet, you probably don't need the `@media` query here.